modules/sff/module/inc/sff/8636.h

```diff
@@ -185,6 +190,17 @@
 #define SFF8636_MEDIA_25GE_CR_N(idprom)                  \
   (idprom[192] == SFF8636_CC192_25GE_CR_N)
 
+#define SFF8636_MEDIA_100GE_LR(idprom)                   \
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...